pkgs/applications/version-management/gitlab/gitaly/default.nix

```diff
@@ -4,6 +4,18 @@
 , libgit2, openssl, zlib, pcre, http-parser }:
 
 let
+  # git2go 32.0.5 does not support libgit2 1.2.0 or 1.3.0.
```

Comment:
Is there a source why this is the case?

Reply:
Well, 1.3.0 resulted in a runtime error, see https://github.com/NixOS/nixpkgs/pull/147848#issuecomment-981628530, 1.2.0 resulted in a link error because `git_remote_name_is_valid` is not exported in libgit2 1.2.0, but git2go needs it.
The specific commit hash was taken from the vendor/libgit2 submodule in the git2go git repo.
The whole thing is a bit too complicated to put in a comment, but it should be fixed once gitaly updates to git2go 33.x, which fully supports libgit2 1.3.0.